src/main/java/sirius/biz/process/SimpleProcessCommand.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.process;
+
+import sirius.biz.process.logs.ProcessLog;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.commons.Values;
+
+import java.util.Map;
+
+/**
+ * Provides the baseline for a simple {@link ProcessCommand} with no arguments.
+ * <p>
+ * Includes boilerplate for running the command in a simulation unless told to actually execute the command.
+ * Implementing classes need to wrap the actual 'real execution' (like db updates/deletes) in an if checking for {@link #shouldExecute(ProcessContext)}.
+ */
+public abstract class SimpleProcessCommand extends ProcessCommand {
+
+    private static final String EXECUTE = "execute";
+
+    protected abstract String createBaseTitle();
+
+    @Override
+    protected String createTitle(Values args) {
+        if (isExecParameterGiven(args)) {
+            return createBaseTitle();
+        }
+        return createBaseTitle() + " (Simulation)";
+    }
+
+    protected abstract String createBaseDescription();
+
+    @Override
+    public String getDescription() {
+        return createBaseDescription() + " Only simulates unless '" + EXECUTE + "' is appended.";
+    }
+
+    protected abstract void executeSimpleProcess(ProcessContext context);
```

Comment:
wouldnt it be nice to pass a boolean parameter whether the execution is a simulation or not?